packages/nimble-components/src/rich-text/models/markdown-serializer.ts

```diff
@@ -9,12 +9,20 @@ import type { Node } from 'prosemirror-model';
  * Provides markdown serializer for rich text components
  */
 export class RichTextMarkdownSerializer {
+    private static mentionedHrefs: string[] = [];
```

Comment:
Similar comment to the parser: we should have tests in the spec for this file to cover the changes to this file.

Reply:
In serializer as all the tests are happening by interactions, Will be adding cases in popup PR as it will have logics for adding mention through interactions. Anyway, I have added cases in editor spec to check the behaviors.